.ci.yaml

```diff
@@ -3695,7 +3677,7 @@ targets:
       - bin/**
       - .ci.yaml
 
-  - name: Mac_arm64_ios module_test_ios # Must be run on devicelab bot for codesigning https://github.com/flutter/flutter/issues/112033
+  - name: Mac_arm64 module_test_ios # Must be run on devicelab bot for codesigning https://github.com/flutter/flutter/issues/112033
```

Comment:
I missed https://github.com/flutter/flutter/issues/112033 when I approved https://github.com/flutter/flutter/pull/141815.  Looks like @vashworth was recently looking at the codesigning issue outside the lab?

Reply:
Yes, so previously a lot of our arm bots were missing certs (https://github.com/flutter/flutter/issues/136651#issuecomment-1771491482), which would cause that test and others to fail so we ran it on devicelab bot. Since upgrading to macOS 13, all Mac-13 chromium bots should have the correct certs now, though. So it should be safe

Reply:
```suggestion
  - name: Mac_arm64 module_test_ios
```

Reply:
Looking at https://github.com/flutter/flutter/issues/112033, we may need to remove these lines to get it to codesign it, though.

https://github.com/flutter/flutter/blob/bae1ac2f6f35a1d49fd826287d5f845e9717eaa2/dev/devicelab/bin/tasks/module_test_ios.dart#L492-L495

Reply:
Actually looks like `mac_arm64 module_test_ios` is failing continuously due to https://github.com/flutter/flutter/issues/112033: https://ci.chromium.org/ui/p/flutter/builders/luci.flutter.staging/Mac_arm64%20module_test_ios. I think we may need to remove [these lines](https://github.com/flutter/flutter/issues/112033#issuecomment-1652268832) so it codesigns. It's weird to me that it needs to be codesigned for a simulator, but only on chromium bot

Reply:
Sounds good, I'll send a separate PR for that change.

Reply:
I think it means in this PR bringup should be removed for `Mac_arm64 module_test_ios` ? And we we should remove `Mac_arm64_ios module_test_ios` and lose coverage until the codesigning issue is addressed?

Reply:
Reverted these tests to its previous state. Would it be possible to get some help landing the fix for the test? that will help me to address a few more PRs to finish updating all the tests.